docs/cli/ilab-local-config.md

```diff
@@ -0,0 +1,184 @@
+# InstructLab CLI Local Storage Proposal
+
+## Why is this needed?
+
+For tools to be useful to the user, they generally need to work on and know about data pertinent to the user. For example, podman and docker track image blobs, kubectl tracks cluster credentials, etc.
+
+The InstructLab CLI manages a number of settings today such as the teacher model, taxonomy location, inference model, etc. using a local file known as `config.yaml`.
```

Comment:
The CLI itself doesn't manage any of this. It's user config.  It generates an initial file for you, but never updates it from there.  I only bring this up because user config isn't the same as data that is never touched directly by a user.  (config vs data)

Reply:
This is a good point, I would say it makes a stronger point for the proposed structure of the `~/.local/share` directory.

Reply:
That might be so @RobotSail but as @russellb mentioned the CLI doesn't manage this.